deps/rabbit/src/rabbit_maintenance.erl

```diff
@@ -180,22 +130,11 @@ is_being_drained_consistent_read(Node) ->
 
 -spec status_local_read(node()) -> maintenance_status().
 status_local_read(Node) ->
-    case catch mnesia:dirty_read(?TABLE, Node) of
-        []  -> ?DEFAULT_STATUS;
-        [#node_maintenance_state{node = Node, status = Status}] ->
-            Status;
-        _   -> ?DEFAULT_STATUS
-    end.
+    rabbit_db_maintenance:get(Node).
```

Comment:
`rabbit_db_maintenance:get/1` can return `undefined` which is not the case of the original `status_local_read/1`. The latter would default to `?DEFAULT_STATUS`.